client/src/components/Settings

```diff
@@ -0,0 +1,189 @@
+                required
+              />
+            </Form.Group>
+            <Form.Group controlId="password" className="mb-3">
+              <Form.Control
+                style={{color: "black"}}
+                type="password"
+                placeholder="Password"
+                name="password"
+                onChange={handleChange}
+                required
+              />
+            </Form.Group>
+            <Form.Group controlId="confirmPassword" className="mb-3">
+              <Form.Control
+                style={{color: "black"}}
+                type="password"
+                placeholder="Confirm Password"
+                name="confirmPassword"
+                onChange={handleChange}
+                required
+              />
+            </Form.Group>
+            <ButtonGroup className="mb-3">
+              <ToggleButton
+                id="has-reports"
+                type="checkbox"
+                variant="primary"
+                checked={formData.hasDirectReports}
+                value="1"
+                onChange={() => toggleDirectReports(true)}
+              >
+                Has Direct Reports
+              </ToggleButton>
+              <ToggleButton
+                id="not-has-reports"
+                type="checkbox"
+                variant="primary"
+                checked={!formData.hasDirectReports}
+                value="2"
+                onChange={() => toggleDirectReports(false)}
+            >
+              Has No Direct Reports
+              </ToggleButton>
+            </ButtonGroup>
+            <Form.Group controlId="managerCode" className="mb-3">
+              <Form.Control
+                style={{color: "black"}}
+                type="text"
+                placeholder="Manager Code"
+                name="managerCode"
+                onChange={handleChange}
+                required
+              />
+            </Form.Group>
+            <Form.Group controlId="positionName" className="mb-3">
+              <Form.Control
+                style={{color: "black"}}
+                type="text"
+                placeholder="Position Name (ex: Builder, Chef, Manager ... )"
+                name="positionName"
+                onChange={handleChange}
+                required
+              />
+            </Form.Group>
+            <Form.Group controlId="companyCode" className="mb-3">
+              <Form.Control
+                style={{color: "black"}}
+                type="text"
+                placeholder="Company Code"
+                name="companyCode"
+                onChange={handleChange}
+                required
+              />
+            </Form.Group>
+            {
+              error &&
+              <p className="error-message">{error}</p>
+            }
+            <Button variant="primary" type="submit" className="w-100">
+                Save
+            </Button>
+          </Form>
+        </div>
```

Comment:
Looking at this form, I believe this is the code for the signin page. This is a great start, we just need to modify this to fit the fields that will be needed in the Settings page(personal information change, theme change, etc). Intitially does not need to be functional, let's focus on getting the UI squared away

Reply:
Yes I used code from the sign in page since in most settings page this is 1:1.

Reply:
Sounds great!